packages/x-components/src/wiring/events.types.ts

```diff
@@ -52,9 +51,11 @@ export interface XEventsTypes
     QuerySuggestionsXEvents,
     RecommendationsXEvents,
     RelatedTagsXEvents,
-    SearchXEvents,
+    ScrollXEvents,
     SearchBoxXEvents,
-    UrlXEvents {
+    SearchXEvents,
+    UrlXEvents,
+    DeviceXEvents {
```

Comment:
You reordered this alphabetically but moved `DeviceXEvents` to the end, why? 🤷🏿‍♂️